src/test/kotlin/net/atos/zac/shared/helper/SuspensionZaakHelperTest.kt

```diff
@@ -104,6 +104,7 @@ class SuspensionZaakHelperTest : BehaviorSpec({
                     opschorting.reden shouldBe postPonementReason
                     einddatumGepland shouldBe null
                     uiterlijkeEinddatumAfdoening shouldBe postponedZaak.uiterlijkeEinddatumAfdoening
+                    isEerderOpgeschort shouldBe true
```

Comment:
I see no tests for the value to be `false`, would be nice to add this...

Reply:
Backend was moved to backend-only PR and is already merged.

With `SuspensionZaakHelperTest` we test the response from suspend operation and we always expect `isEerderOpgeschort` to be true.

The `false` branch is tested from `ZaakRestServiceTest` integration test.